Main.java

```diff
@@ -9,7 +11,23 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // create a Scanner for user input 
+    Scanner input = new Scanner(System.in);
+
+    // ask the user for two numbers
+    System.out.println("Please enter an integer:");
+    int firstInt = input.nextInt();
+
+    System.out.println("Please enter another integer:");
+    int secondInt = input.nextInt();
+
+    if (firstInt > secondInt) {
+      System.out.println("The biggest number is " + firstInt + ".");
+    } else if (firstInt == secondInt) {
+      System.out.println("There is no biggest number.");
+    } else {
+      System.out.println("The biggest number is " + secondInt + ".");
```

Comment:
Make sure that you are adding comments to your code. Think of the steps your program makes and turn these into comments. Place them above the code that they represent :)